chapter16/worker/worker.js

```diff
@@ -1,167 +1,134 @@
-var express = require('express');
-var bodyParser = require('body-parser');
-var AWS = require('aws-sdk');
-var assert = require('assert-plus');
-var Jimp = require('jimp');
-var fs = require('fs');
+const AWS = require('aws-sdk');
+const assert = require('assert-plus');
+const Jimp = require('jimp');
+const fs = require('fs/promises');
 
-var lib = require('./lib.js');
+const lib = require('./lib.js');
 
-var db = new AWS.DynamoDB({
+const db = new AWS.DynamoDB({
   'region': 'us-east-1'
 });
-var s3 = new AWS.S3({
+const s3 = new AWS.S3({
+  'region': 'us-east-1'
```

Comment:
is setting the region still needed on ec2? i might be wrong but it could work without it (via IMDS).